### PR TITLE
Teslaloose and Singuloose counterplay

### PR DESCRIPTION
**Pull request by TheShuEd**
## About the PR

Adds 2 new items, in the T2 experimental research branch.

- force field holoprojector. create a 3-minute barrier that doesn't let anything through, including the singularity and tesla
- portable particle decelerator. A two-handed heavy device that shoots out negative PA particles. These particles kill the tesla and the singularity.

Given the slow rate of fire, and the ability of thesla and singularity to gain energy by absorbing objects, more than 1 shooter of this cannon will likely be needed.

expected gameplay - a large number of players must cooperate to trap a tesla or singularity and group bombard it with negative particles.

Unfortunately, I don't have the opportunity to test the complexity and balance of these devices alone. Therefore, I may need to make some parameter adjustments in the future.

## Why / Balance

players get to resist the end of the round in the event of a tesla or singularity escape.

## Media

https://github.com/space-wizards/space-station-14/assets/96445749/6d59e76a-e51b-486f-b723-a8fe341bea13

image
image

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase
**Changelog**

:cl:
- add: Added new T2 "Deterrence technologies", it includes 2 new tools to fight a runaway singularity or tesla. Force barrier holoprojector that sets up a 3-minute force containment field, and portable particle decelerator, capable of damaging a tesla or singularity. Chances are 1 gun will not be enough! You will need a teamwork of several shooters and hunters to destroy a tesla or singularity.
- add: Added in cargo "Particle Decelerators crate" with 3 PD, at cost 15000$

### CODE REVIEW

**Comment by Ilya246** [comment]
cargo should be able to buy the decelerator at a high price (5k per?)

**Comment by FairlySadPanda** [comment]
Could this gun also be used to kill anoms?

**Comment by TheShuEd** [comment]
cargo is already ordering a lot of things to build the singularity or tesla.  Let's leave the issue of containment to rnd to increase communications between departments

**Comment by lzk228** [comment]
emag portable accelerator to shoot positive particles :trollface:

**Comment by TheShuEd** [comment]
no, it does not cause damage to anomalies, just as ordinary particles do not grow it.  But this is an interesting idea that could be implemented in the future!

**Comment by TheShuEd** [comment]
emag is a cool idea.  I'll remember it for the future

**Comment by Ilya246** [comment]
> cargo is already ordering a lot of things to build the singularity or tesla. Let's leave the issue of containment to rnd to increase communications between departments

there's no way sci can make the containment stuff in time if there's a loose and i don't think sci would bother to research and make them otherwise, so the gun should be orderable at cargo so you can actually get it without having to rely on sci

**Comment by TheShuEd** [comment]
Let's wait what Emo says.  he suggested adding it to T2

**Comment by Ilya246** [comment]
> Let's wait what Emo says. he suggested adding it to T2

i'm saying both as t2 sci and an expensive emergency cargo purchase

**Comment by TheShuEd** [comment]
Ok

**Comment by Kadeo64** [comment]
couple notes:
1: did I just see the particle decelerator be crafted in an autolathe?
2: why tier 2 experimental, it's an engineering obj- you know what? fuck tier 2 industrial. 
3: force field seems useful for more than just tesla/singulo

**Comment by EmoGarbage404** [comment]
Worth stating for posterity:

Experimental =/= science equipment
Industrial =/= engineering equipment

Also i think having the particle decelerator in cargo is probably okay since the tech also has the force field stuff.

**Comment by TheShuEd** [comment]
so should I move it to industrial?

**Comment by EmoGarbage404** [comment]
I think its fine in experimental. Maybe it could even be in arsenal

**Comment by Ilya246** [comment]
also it should do something to people if it hits them

**Comment by Errant-4** [comment]
A 3 minute impassable barrier with no counterplay apart from "find another route" sounds very powerful for uses unrelated to containing the singularity

**Comment by TheShuEd** [comment]
I agree, I’ll add destructibility to it from physical damage.  I also noticed a mistake that the gun can be hung on a belt

**Comment by K-Dynamic** [comment]
TARCOL MINTI ZHERI!

I like both of these but forcefield sprite looks too much like holobarrier

**Comment by TheShuEd** [comment]
Well, essentially this is a holobarrier, just a power force one.  Perhaps he should add some sparks?

**Comment by Hmeister-fake** [comment]
Are you able to jump over the barrier? Like the sec one?

**Comment by TheShuEd** [comment]
No

**Comment by K-Dynamic** [comment]
> No

Okay the sprite really needs a change to avoid confusion since:
A) You can't jump over it
B) It's designed to contain looses

**Comment by TheShuEd** [comment]
https://github.com/space-wizards/space-station-14/assets/96445749/0a436ffb-11d0-4d51-8feb-7fd20e3e5748

resprite field

File: Resources/Locale/en-US/research/technologies.ftl [hunk 1]
```diff
@@ -59,6 +59,7 @@ research-technology-gravity-manipulation = Gravity Manipulation
 research-technology-advanced-anomaly-research = Advanced Anomaly Research
 research-technology-rped = Rapid Part Exchange
 research-technology-super-parts = Super Parts
+research-technology-deterrence = Deterrence technologies
```
Comment (mirrorcult) [inline Resources/Locale/en-US/research/technologies.ftl:62]:
```suggestion
research-technology-deterrence = Deterrence Technologies
```

i think there could be a better name for this but eh

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml [hunk 2]
```diff
@@ -313,6 +313,37 @@
     proto: RedHeavyLaser
     fireCost: 100
 
+- type: entity
+  name: portable Particle Decelerator
```
Comment (mirrorcult) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml:317]:
```suggestion
  name: portable particle decelerator
```

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml [hunk 3]
```diff
@@ -313,6 +313,37 @@
...
+  parent: BaseWeaponBattery
+  id: WeaponParticleDecelerator
+  description: A portable particle decelerator capable of decomposing a tesla or singularity
```
Comment (mirrorcult) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml:320]:
```suggestion
  description: A portable particle decelerator capable of decomposing a tesla or singularity.
```

**Review by mirrorcult — APPROVED** [review]

**Pull request merged**